lib/matplotlib/backends/backend_agg.py

```diff
@@ -485,6 +485,42 @@ def print_raw(self, filename_or_obj, *args, **kwargs):
     print_rgba = print_raw
 
     def print_png(self, filename_or_obj, *args, **kwargs):
+        """
+        Write the png data to the given file.
+
+        Parameters
+        ----------
+        filename_or_obj : str or PathLike or file-like object
+            The file to write to.
+
+        metadata : dict, optional
+            Metadata in the PNG file as key-value pairs of strings.
```

Comment:
Looking at the implementation of _png.cpp, looks like this should actually be "of latin-1-encodable strs or of bytes".  The relevant chunk:
```
            if (PyUnicode_Check(meta_key)) {
                PyObject *temp_key = PyUnicode_AsEncodedString(meta_key, "latin_1", "strict");
                if (temp_key != NULL) {
                    text[meta_pos].key = PyBytes_AsString(temp_key);
                }
            } else if (PyBytes_Check(meta_key)) {
                text[meta_pos].key = PyBytes_AsString(meta_key);
```